linkerd/app/inbound/src/http.rs

```diff
@@ -74,7 +74,14 @@ pub mod fuzz {
                             .header(header_name, header_value)
                             .body(Body::default())
                         {
-                            let rsp = http_util::http_request(&mut client, req).await;
+                            let rsp = client
+                                .ready()
+                                .await
+                                .expect("HTTP client poll_ready failed")
+                                .call(req)
+                                .await
```

Comment:
tioli: you can use [ServiceExt::oneshot](https://docs.rs/tower/latest/tower/trait.ServiceExt.html#method.oneshot) to make all of these calls `client.clone().oneshot(req).await.expect("...")`

Reply:
this didn't seem to compile unfortunately, because the `SendRequest` type isn't cloneable.

that said, none of these tests _need_ it to be, because they only send a single request. so, https://github.com/linkerd/linkerd2-proxy/pull/3428/commits/d16ccc0f6e50e7ae09673fc6036fba3cd33be2d0 calls `oneshot()`, sans cloning.

this looks much better, thank you for suggesting this :slightly_smiling_face: